content/applications/marketing/survey/survey_essentials.rst

```diff
@@ -0,0 +1,301 @@
+
+.. image:: survey_essentials/live-session-setting.png
+   :align: center
+   :alt: live session section of survey features
+
+Test & share
+============
+
+Once your survey is saved and ready, you can test it to check for possible errors before sending it
+out to your participants.
+
+Simply click *Test* in the upper left corner of the survey template page.
+
+.. image:: survey_essentials/test-survey-button.png
+   :align: center
+   :alt: test smart button on survey template form
+
+When clicked, Odoo takes you to survey on the front-end of the website, and shows you how the
+survey looks to participants. You can then proceed to run through the survey, like a normal
+participant, to check for any errors.
+
+Along the top of the page, there's a blue banner, signifying that this is a test survey. To return
+to the survey template form on the backend, click that link in the blue banner.
+
+.. image:: survey_essentials/blue-banner-test.png
+   :align: center
+   :alt: blue banner on test survey with link
+
+Once there, you can make any additional changes or modifications that you'd like, before sending it
+out to potential participants.
+
+Then, when you're satisfied with how the survey looks, you can click *Start Survey* in the
+upper-left corner.
+
+.. image:: survey_essentials/start-survey-button.png
+   :align: center
+   :alt: start survey smart button on survey template form
+
+When clicked, the status of the survey moves from *Draft* to *In Progress*, and more buttons appear.
+
+.. image:: survey_essentials/survey-status-bar.png
+   :align: center
+   :alt: survey status bar from draft to in progress
+
+* *Share* - email potential participants a link and invitation to the survey
+* *See Results* - provides a visual representation of all the answers so far
+* *Create Live Session* - starts a Live Session, and provides a specific URL for participants
+* *Test* - test out the survey to check for errors
+* *Print* - print out the survey
+* *Close* - officially close the survey to participants
+
+.. image:: survey_essentials/new-survey-button-options.png
+   :align: center
+   :alt: new buttons present on survey template form
+
+Also, as answers get collected, you can check them by clicking the *Answers* smart button on the
+survey template form.
+
+.. image:: survey_essentials/survey-answers-button.png
+   :align: center
+   :alt: answers smart button on survey template
+
+You can also access the detailed answers by going to
+:menuselection:`Participations --> Participations`, as well.
+
+.. image:: survey_essentials/participations.png
+   :align: center
+   :alt: the participations drop-down menu with participations
+
+Once there, simply open up the survey you wish to examine, and click on any answer form to gain
+more information.
+
+.. image:: survey_essentials/survey-participations-page.png
+   :align: center
+   :alt: survey drop-down open on participations page
+
+.. seealso::
+    - :doc:`scoring`
+    - :doc:`time_random`
```

Comment:
```suggestion
```
This is too basic overall unless you want to go deeper into one of the topics you present here. To remove as it is.